geonode_mapstore_client/client/js/api/geonode/v2/index.js

```diff
@@ -406,6 +419,113 @@ export const getResourcesTotalCount = () => {
         });
 };
 
+export const getCategories = ({ q, idIn, ...params }, filterKey = 'categories') => {
+    return axios.get(parseDevHostname(`${endpoints[CATEGORIES]}`), {
+        params: {
+            limit: 9999,
+            ...params,
+            ...(idIn && {'filter{identifier.in}': idIn}),
+            ...(q && { 'filter{identifier.contains}': q })
```

Comment:
we were using `icontains` with the old api, could we use the same with the new one?
Please replace `contains` with `icontains` if possible